components/movementsensor/gpsrtkpmtk/gpsrtkpmtk.go

```diff
@@ -354,6 +355,15 @@ func (g *rtkI2C) receiveAndWriteI2C(ctx context.Context) {
 		g.logger.Infof("caster %s seems to be down", g.ntripClient.URL)
 	}
 
+	srctable, err := g.ntripClient.Client.ParseSourcetable()
+	if err != nil {
+		g.logger.Errorf("failed to get source table: %v", err)
```

Comment:
Naive question from someone who doesn't know Go well: shouldn't you use `%w` to wrap an error? When do you use that, and when `%v`?

Reply:
`%v` is the styling for anything when you're trying to log, rather than `%w` which is specifically for an error. 

I do like `%v` since it takes the load off the variable changing in the future, or thinking a variable is one type and then figuring out through your logs after it is in production that it is in fact another.  But your mileage may vary on using the appropriate wrapper when you know for sure the wrapper is a specific type, as in this case where we know for sure it is an error.